db/seeds.rb

```diff
@@ -5,39 +5,165 @@
   password: 'password'
 )
 
-puts 'creating views...'
-about_us_view = View.create(name: 'about_us', variant: 1)
-about_self_care_view = View.create(name: 'about_self_care', variant: 1)
-information_view = View.create(name: 'information', variant: 1)
-services_view = View.create(name: 'services', variant: 0)
-
-puts 'creating image...'
-image_file = URI.open('https://assets.zyto.com/wp-content/uploads/2018/05/holistic-health-concept-woman-doing-yoga-on-beach-at-dusk.jpg')
-image = Image.create(alt_text: 'Alt attribute')
-image.file.attach(io: image_file, filename: 'article_image.jpg', content_type: 'image/jpg')
-
-puts 'creating sections...'
-section_regular = Section.create(
-  view_id: information_view.id,
-  header: 'Section regular',
-  description: 'A Solid App is composed of functions that we call Components. Component names in Solid follow the Pascal naming convention in which the first letter of each word in a component name is capitalized. This name can then be used as tags within our JSX like',
-  variant: 0,
-  image: image
-)
-section_no_image = Section.create(
-  view_id: about_self_care_view.id,
-  header: 'Section no Image',
-  description: 'A Solid App is composed of functions that we call Components. Component names in Solid follow the Pascal naming convention in which the first letter of each word in a component name is capitalized. This name can then be used as tags within our JSX like',
+# ---------- CREATING SECTONS FOR ABOUT US VIEW -----------
```

Comment:
Before we start seeding we have to make sure that we clear all data.(I commented on this on Slack) I suggest something like

```ruby
[Section, Cta, View].each do | model |
  model.destroy_all
end
```

Reply:
added